tasks/2017/interactive-bots.md

```diff
@@ -41,34 +43,35 @@ Follow the tutorial below to test run the helloworld bot on your Zulip
 instance (either local or in your droplet).  As you're going through the tutorial,
 take notes on where you got stuck or needed to look up terminology.
 
-Refer to the detailed description on how to [run a zulip bot](https://chat.zulip.org/api/running-bots).
+Follow the steps below to run the helloworld bot:
 
-Follow the steps below to run helloworld bot:
+* Go through the steps outlined in the [Running a bot guide](
```

Comment:
Do we want them to test against their development environment?  If so, we might want to point them at a zulipdev.com URL (or whatever) rather than chat.zulip.org; otherwise, after getting their zuliprc, they'll probably do it against chat.zulip.org by accident.